README.md

```diff
@@ -373,16 +385,21 @@ Otherwise, they will be resent at the next interval.
     >>
     >> [
     >>   {
-    >>     topic: <name>,
-    >>     type:  <type>,
-    >>     url:   <url>,
-    >>     t:     <t>
+    >>     topic: <string>,
+    >>     type:  <string>,
+    >>     url:   <string>,
+    >>     t:     <integer>,
+    >>     data:  <anything>
```

Comment:
Well, a string is valid JSON. The code here doesn't assume a JSON object, for instance — the data blob is opaque.